pkg/network/http/shared_libraries.go

```diff
@@ -84,7 +85,7 @@ func (w *soWatcher) Start() {
 	go func() {
 		ticker := time.NewTicker(soSyncInterval)
 		defer ticker.Stop()
-		thisPID := os.Getpid()
+		thisPID := util.GetRootNSPID()
```

Comment:
This will return `0` in the error case. Is there a danger of a loop in that case, since we won't be filtering any events?

Reply:
That's fine. We may resolve certain paths twice, but it's just a small perfomance hit and don't change the semantics of the program.